PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -382,18 +382,24 @@ DECLARE_SOA_TABLE(HfCandBpConfigs, "AOD", "HFCANDBPCONFIG", //! Table with confi
 // Charm resonances analysis
 namespace hf_reso_cand_reduced
 {
-DECLARE_SOA_COLUMN(InvMass, invMass, float);                     //! Invariant mass of 3 prong candidate in GeV/c2
+DECLARE_SOA_COLUMN(InvMass, invMass, float);                     //! Invariant mass in GeV/c2
 DECLARE_SOA_COLUMN(InvMassK0s, invMassK0s, float);               //! Invariant mass of V0 candidate in GeV/c2, under K0s mass assumption
 DECLARE_SOA_COLUMN(InvMassLambda, invMassLambda, float);         //! Invariant mass of V0 candidate in GeV/c2, under Lambda mass assumption
 DECLARE_SOA_COLUMN(InvMassAntiLambda, invMassAntiLambda, float); //! Invariant mass of V0 candidate in GeV/c2, under AntiLambda mass assumption
 DECLARE_SOA_COLUMN(Px, px, float);                               //! Momentum of V0/3 prong candidate in GeV/c
 DECLARE_SOA_COLUMN(Py, py, float);
 DECLARE_SOA_COLUMN(Pz, pz, float);
-DECLARE_SOA_COLUMN(CosP, cosp, float);       //! Cos PA of V0 candidate
-DECLARE_SOA_COLUMN(DCA, dca, float);         //! DCA of V0 candidate
+DECLARE_SOA_COLUMN(Cpa, cpa, float);         //! Cosine of Pointing Angle of V0 candidate
+DECLARE_SOA_COLUMN(Dca, dca, float);         //! DCA of V0 candidate
 DECLARE_SOA_COLUMN(Radius, radius, float);   //! Radius of V0 candidate
 DECLARE_SOA_COLUMN(V0Type, v0Type, uint8_t); //! Bitmap with mass hypothesis of the V0
 DECLARE_SOA_COLUMN(DType, dType, int8_t);    //! Integer with selected D candidate type: 1 = Dplus, -1 = Dminus, 2 = DstarPlus, -2 = DstarMinus
```

Comment:
For clarity maybe this can be renamed as `sign`

Reply:
OK! Then should I remove the distinction between D+ and D* or add it in a separate column?

Reply:
No, you are right, I overlooked it. Good as it is since we combine the info of particle species and sign, thanks!